### ISSUE TITLE
Accessibility is missing

### ISSUE THREAD

**Issue by Koraiko** [Koraiko/cv-webpage#10]
# Description
Accessibility labels are missing (i.e. aria-labels). Maybe theres something else to make the website more accessible?

# ToDo
- [ ] add aria labels to every important item
- [ ] research how to make the website more accessible.
